### DISCUSSION TITLE
[ Feature request ] Support zsh integration

### DISCUSSION THREAD

**Discussion by LinArcX** [microsoft/vcpkg#17742] [Ideas]

**Comment by LilyWangL** [comment]
Thanks for posting this issue. Can you provide more information about zsh integration?

**Comment by LinArcX** [comment]
@wangli28 Hi. look at this:
https://github.com/microsoft/vcpkg#tab-completion--auto-completion

**Comment by bpoebiapl** [comment]
please support zsh auto completion
![image](https://user-images.githubusercontent.com/18179088/66170864-f0b35e00-e680-11e9-98db-115082e61cde.png)

**Comment by gsfreema** [comment]
It isn't mentioned here, but macOS Catalina switched the default shell from bash to zsh so having integration for zsh would be very useful for macOS users.

**Comment by ghost** [comment]
Any update about this as of today?

**Comment by LilyWangL** [comment]
@ras0219-msft Could you help take a look this issue?

**Comment by stardiviner** [comment]
Any update? Still waiting for this feature.

**Reply by PhoebeHui** [reply]
Thanks @autoantwort to support this in vcpkg, it would be avaliable in next release https://github.com/microsoft/vcpkg-tool/pull/87#issuecomment-887875694

**Reply by stardiviner** [reply]
That's great!
